### PR TITLE
Initial cut at `rly tx pkt`

### PR DESCRIPTION
**Pull request by michaelfig**
This now works in general.  Have a look at `rly tx send-packet --help` for details.

```
soil:relayer michael$ rly tx send-packet -d ibc0 ibc1 'hello from ibc0!'
I[2020-04-20|23:17:11.747] ✔ [ibc0]@{6197} - msg(0:sendpacket) hash(B8461AF5F4B8AE55E558391CF0192956ED9AFA5D180B2D83EA9DE4719D3A9FD3) 
soil:relayer michael$ rly start ibczeroibcone
I[2020-04-20|23:18:12.659] - listening to tx events from ibc0...        
I[2020-04-20|23:18:12.659] - listening to block events from ibc0...     
I[2020-04-20|23:18:12.660] - listening to tx events from ibc1...        
I[2020-04-20|23:18:12.660] - listening to block events from ibc1...     
I[2020-04-20|23:18:13.686] ✔ [ibc1]@{6255} - msg(0:update_client,1:ics04/opaque) hash(5BD3DB812B0C2A8E06257C0E98E62A2FAAE82908DE79F26751E4B283D6234B81) 
I[2020-04-20|23:18:13.686] ★ Relayed 1 packets: [ibc0]port{echo}->[ibc1]port{echo} 
I[2020-04-20|23:18:13.687] • [ibc1]@{6255} - actions(0:update_client,1:ics04/opaque) hash(5BD3DB812B0C2A8E06257C0E98E62A2FAAE82908DE79F26751E4B283D6234B81) 
^CSignal Recieved: interrupt
I[2020-04-20|23:18:16.862] - [ibc0]:{echo} <-> [ibc1]:{echo} relayer shutting down 
soil:relayer michael$ 
```

### CODE REVIEW

**Comment by michaelfig** [comment]
Hi @jackzampolin, this is ready for you to have a look.  It would be great to land, but please let me know if you have any changes (such as removing the `swingset` dependencies).

**Comment by jackzampolin** [comment]
Will take a peek tomorrow morning, but don't mind the swingset dep

File: relayer/packetMsgs.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	chanTypes "github.com/cosmos/cosmos-sdk/x/ibc/04-channel/types"
+)
+
+var ModuleCdc *codec.Codec
+
+func RegisterCodec(cdc *codec.Codec) {
```
Comment (jackzampolin) [inline relayer/packetMsgs.go:14]: Shouldn't this live in the Agoric Chain?

**Review by jackzampolin — APPROVED** [review]
:+1: on the approach. Only question is if that packet type should live in a chain (i.e. `swingset` or `cosmos-sdk`).

File: relayer/packetMsgs.go (hunk 1 above)
Comment (michaelfig) [reply relayer/packetMsgs.go:14]: If there's a way of encoding these packets without using `packetMsgs.go` or depending on third-party stuff, I'm all for it.

Maybe the Route doesn't need to be set?

**Comment by michaelfig** [comment]
> 👍 on the approach. Only question is if that packet type should live in a chain (i.e. `swingset` or `cosmos-sdk`).

Can you explain what you have in mind?  I didn't want to introduce a dependency from the relayer on Agoric, since we haven't yet published our Go modules.

**Review by jackzampolin — APPROVED** [review]
Thanks @michaelfig! I'm going to take a stab at pushing up an implementation of SendPacket into the SDK.

**Pull request merged**